conf/application.conf

```diff
@@ -19,56 +19,18 @@ include "backend.conf"
 
 appName = trusts-individual-check-stub
 
-# Provides an implementation of AuditConnector. Use `uk.gov.hmrc.play.audit.AuditModule` or create your own.
-# An audit connector must be provided.
-play.modules.enabled += "uk.gov.hmrc.play.audit.AuditModule"
-
-# Provides an implementation of MetricsFilter. Use `uk.gov.hmrc.play.graphite.GraphiteMetricsModule` or create your own.
-# A metric filter must be provided
-play.modules.enabled += "uk.gov.hmrc.play.bootstrap.graphite.GraphiteMetricsModule"
-
-# Provides an implementation and configures all filters required by a Platform frontend microservice.
-play.modules.enabled += "uk.gov.hmrc.play.bootstrap.backend.BackendModule"
-
 # Default http client
 play.modules.enabled += "uk.gov.hmrc.play.bootstrap.HttpClientModule"
 
-# Json error handler
-play.http.errorHandler = "uk.gov.hmrc.play.bootstrap.backend.http.JsonErrorHandler"
```

Comment:
not entirely sure you can delete this here?

Reply:
oh I see it here https://github.com/hmrc/bootstrap-play/blob/main/bootstrap-backend-play-30/src/main/resources/backend.conf#L42 , maybe to leave it here until B&D implement pr bot commenter rule and ti will reported like the rest of them?

Reply:
I have been doing it for a while on other repos but can add back - i feel like if we update bootstrap again though and it is required to be back, it will be clear ?

Reply:
the reason why i asked is because https://github.com/hmrc/init-service/blob/main/init_service/templates/service/conf/application.conf#L58 has this line in place

Reply:
im ok to delete it here as it is a stub but wanted to kn ow the reason